requirements/skills-media.txt

```diff
@@ -1,6 +1,5 @@
 # skills for OCP, require audio playback plugins (usually mpv)
-ovos-skill-somafm>=0.0.2,<1.0.0
-skill-news>=0.0.4,<1.0.0
+ovos-skill-somafm>=0.1.1,<1.0.0
+skill-news>=0.1.2,<1.0.0
 ovos-skill-pyradios>=0.1.0,<1.0.0
-ovos-skill-local-media>=0.2.0,<1.0.0
-ovos-skill-youtube-music>=0.1.1,<1.0.0
```

Comment:
removed youtube-music because youtube is unstable and keeps breaking, let users decide if they are up to that...